src/main/java/org/mastodon/mamut/tomancak/sort_tree/SortTree.java

```diff
@@ -60,6 +60,11 @@ public static void sortExternIntern( Model model, Collection<Spot> vertices, Col
 		sort( model, vertices, new ExternInternOrder( model.getGraph(), centerSpots ) );
 	}
 
+	public static void sortCellLifetime( Model model, Collection<Spot> vertices )
```

Comment:
It would be good for future non-familiar developers to have some javadoc explaining what this public method does. I think it cannot be straightforward be derived from method signature, without further knowledge. An example visualizing the result would be good, cf. my suggestion:

```suggestion
/**
	 * <pre>
	 *          ┌───────────┐               ┌───────────┐
	 *          │3          │2              │2          │3
	 *          │           │               │           │
	 *          │        ┌──┘──┐     >   ┌──┘──┐        │
	 *       ┌──┘──┐     │1    │2        │1    │2    ┌──┘──┐
	 *       │2    │1          │               │     │1    │2
	 *       │                                             │
	 *</pre>
	 *
	 */
```

Reply:
I added a similar javadoc.

The order is actually the opposite of what you showed there.